src/main/kotlin/com/msd/command/application/CommandApplicationService.kt

```diff
@@ -127,7 +127,7 @@ class CommandApplicationService {
                 RepairItemType.valueOf(args[1].uppercase()),
                 UUID.fromString(args[2])
             )
-            else -> throw RuntimeException("Internal Error")
+            else -> throw RuntimeException("Unknown command verb $verb")
```

Comment:
I think it would be nicer if we used a custom exception here

Reply:
This actually cannot be thrown, but I'll change it anyways